test/queryplan/queryPlanZoom.html

```diff
@@ -0,0 +1,293 @@
+
+			var graph = JSON.parse(s);
+
+			var imageBasePath = './icons/';
+			var iconPaths =
```

Comment:
we should probably refactor the boilerplate icon/sample graph data into a shared JS file to avoid duplication in every test HTML file at some point.